OpenRA.Game/Network/GameServer.cs

```diff
@@ -132,6 +132,9 @@ public class GameServer
 		[FieldLoader.LoadUsing("LoadClients")]
 		public readonly GameClient[] Clients;
 
+		/// <summary>The list of spawnpoints that are disabled for this game</summary>
+		public readonly int[] DisabledSpawnPoints;
```

Comment:
Please initialize this as `public readonly int[] DisabledSpawnPoints = { };`. Otherwise, the game crashes when selecting a server in the multiplayer browser.